api/locale/ur.yml

```
@@ -0,0 +1,21 @@
+ur:
+  direction: rtl
+  views: "%{number} ویوز"
```

Looks great! :+1: 

I noticed on YouTube, the Urdu translation for views appears to be a little different.

eg. [image](https://user-images.githubusercontent.com/20955511/196015856-c494c248-e5dd-4b9b-8b74-ca76349dd8f1.png)

I am not as familiar with the language as I'm sure you are, so let me know if you think this translation would also be accurate.

```suggestion
  views: "%{number} ملاحظات"
```

---

Oh I actually was going for that 2nd word but it's actually kind of an advanced Urdu word which I thought wouldn't be suitable. But if the youtube app is using it I think it would be better to use that version. I'll commit this suggestion. 👌